### ISSUE TITLE
Play silence to prevent missing information at start of speech

### ISSUE THREAD

**Issue by thadeu220** [nvaccess/nvda#14386]
### Is your feature request related to a problem? Please describe.
When using some bluetooth headphones, speech output is not heard after NVDA has some moments of silence.
The headphones appear to go into standby and miss the start of speech.
Keeping NVDA active is tiring, missing speech is annoying and confusing.

### Describe the solution you'd like
Add an option to play silence to keep NVDA active.

### Describe alternatives you've considered
Use [bluetooth audio add-on](https://addons.nvda-project.org/addons/BluetoothAudio.en.html).

### Additional context
- This is a problem that is likely to be common to many users, bluetooth headsets / airpods are very common.
- This feature is available in Jaws since 2020.

**Comment by mazen428** [comment]
Look at the  [bluetooth audio add-on](https://addons.nvda-project.org/addons/BluetoothAudio.en.html)

**Comment by Brian1Gaff** [comment]
There used to be a stand alone exe that was called something like 
Silencio.exe that did this to get over the latency in many bits of sound 
hardware and their drivers by a well known driver maker. Maybe this is what 
is required here? It depends on how the latency occurs on the headphones, 
and how they detect there is a a sound.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message ----- 
From: "thadeu220" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Sunday, November 20, 2022 12:32 AM
Subject: [nvaccess/nvda] suggestion: add a silence sound to avoid latency on 
wireless headphones via bluetooth (Issue #14386)

**Comment by jcsteh** [comment]
My pull request was to keep the audio device open for longer. That got rewritten somewhat by NV Access, but the core idea got merged already. However, it seems that for some devices, even keeping the device open isn't enough and you actually have to play silence. I never submitted a pull request for that, but the Bluetooth Audio add-on does this.

**Comment by cary-rowen** [comment]
Would love to see this proposal implemented

**Comment by Adriani90** [comment]
I wonder if #14697 will have any positive effect on this?

**Comment by jcsteh** [comment]
Unlikely, but I am curious nevertheless. That said, it should be easier to implement continuous silence with the WASAPI code.
